### PR TITLE
feat/add responsive design for new button

### LINKED ISSUE #4: Setup Responsiveness

### LINKED ISSUE #23: Responsive Design

### PR DESCRIPTION
**Pull request by tenthwalker**
What I did:
This pull request removes a few unnecessary comments, consolidates a few lines of CSS, removes unnecessary files, and fixes an id/class inconsistency from a prior merge.

Did this break anything?

- [x] No
- [ ]  Yes

Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Testing

### CODE REVIEW

**Comment by JoeyStewart** [comment]
The buttons are way better looking.

**Review by Jesuitman — APPROVED** [review]

**Comment by MaryBruff** [comment]
Looks soooo good! The responsive is brilliant!

File: src/App/App.js [hunk 1]
```diff
@@ -16,15 +16,14 @@ function App() {
   const [savedControversies, setSavedControversies] = useState([]);
 
   
-  const randomSearchInputs = ['4chan', 'Titan (Submersible)', 'Billy Mitchell (gamer)', 'You Showed Me', 'Ezra Miller', 'Russel Brand', 'Bernie Madoff', 'Amy Winehouse']
+  const randomSearchInputs = ['4chan', 'Titan (Submersible)', 'Billy Mitchell (gamer)', 'You Showed Me', 'Ezra Miller', 'Russell Brand', 'Bernie Madoff', 'Amy Winehouse']
```
Comment (MaryBruff) [inline src/App/App.js:19]: Finally! He lives with two LLs

**Comment by Jesuitman** [comment]
awesome job on this! looks great!

File: src/App/App.css [hunk 2]
```diff
@@ -63,3 +63,12 @@ button:hover {
...
 }
+
+@media screen and (max-width: 500px) {
+  h1 {
+    font-size: 3rem;
+  }
+  .App-header button {
+    font-size: .5em;
+  }
+}
```
Comment (JoeyStewart) [inline src/App/App.css:74]: Its a responsive behemoth

**Review by JoeyStewart — APPROVED** [review]

**Pull request merged**